File: services/servico-order/src/domain/OrderStatus.ts
/**
 * Estados do ciclo de vida do pedido (Order), conforme o caso de uso
 * "Realizar pedido" descrito na Macroatividade 2 (EC-M2.2) do estudo de caso.
 */
export enum OrderStatus {
  AGUARDANDO_ACEITACAO = 'AGUARDANDO_ACEITACAO',
  PREPARANDO = 'PREPARANDO',
  AGUARDANDO_ENTREGA = 'AGUARDANDO_ENTREGA',
  ENTREGUE = 'ENTREGUE',
  CANCELADO = 'CANCELADO',
}

File: services-java/servico-order/src/main/java/br/com/ftgo/order/domain/OrderStatus.java
package br.com.ftgo.order.domain;

/**
 * Estados do ciclo de vida do pedido (Order), conforme o caso de uso
 * "Realizar pedido" descrito na Macroatividade 2 (EC-M2.2) do estudo
 * de caso.
 */
public enum OrderStatus {
    AGUARDANDO_ACEITACAO,
    PREPARANDO,
    AGUARDANDO_ENTREGA,
    ENTREGUE,
    CANCELADO
}
